### DISCUSSION TITLE
Local Order Parameter

### DISCUSSION THREAD

**Discussion by jmarbues** [brucefan1983/GPUMD#785] [Q&A] [answered]
Hi @brucefan1983. In issue #385, someone asked about the possibility of implementing on-the-fly computation of local order parameters, and you said that you would aim to add this feature to GPUMD-v3.9.

If you still intend to add this to GPUMD, could you give us the expected release date?

Thank you very much for your time.
Kind regards, José

**Comment by brucefan1983** [comment, marked as answer]
Ah apparently I failed the goal...

I estimate it will be completed between Jan and March 2025.

**Reply by jmarbues** [reply]
Ok thank you
